### DISCUSSION TITLE
No new release until January

### DISCUSSION THREAD

**Discussion by jsjoeio** [coder/code-server#5918] [Announcements]
**:no_good: no additional releases this month**
While testing code-server after the Code 1.74 upgrade, we noticed extensions not working. Turns out there's a :bug: upstream. Therefore, instead of doing a release candidate only with a buggy code-server, we decided to hold off until this bug is fixed (VS Code Team said 1.75 should have it).

See these issues for more details:
- [Extensions in portable mode are broken](https://github.com/microsoft/vscode/issues/169006)
- [Latest VSCode (1.74.2) with local ./data folder no longer works when renaming the root folder](https://github.com/microsoft/vscode/issues/169714)

The maintainers will be offline between now and January 2nd (back Jan. 3rd) as well. 

Happy holidays! ❄️

**Comment by filiptronicek** [comment]
#NoDecemberReleaseGang 😄 

happy holidays, @jsjoeio!

**Comment by lg-nineoneninethreenine** [comment]
Happy holidays!

**Comment by arnabdebnath208** [comment]
Holiday sweet holidays
